daemon/src/rollover_maker.rs

```diff
@@ -137,7 +139,8 @@ impl Actor {
         let order_id = self.order_id;
         let AcceptRollover {
             tx_fee_rate,
-            funding_rate,
+            long_funding_rate,
+            short_funding_rate,
```

Comment:
note that we are paying a price by using the "command" abstraction here - we are passing an unused funding rate, which can be confusing to the users.
Perhaps we could document this in `AcceptRollover`? (that only one of them gets picked, depending on position)

On the plus side, we are removing one additional cfd load, which is good. :-)

Reply:
> Perhaps we could document this in `AcceptRollover`? (that only one of them gets picked, depending on position)

Yes can document that!